src/features/search/components/__tests__/useLocationChoice.test.ts

```diff
@@ -0,0 +1,90 @@
+import { useLocationChoice } from 'features/search/components/useLocationChoice'
+import { LocationType } from 'features/search/enums'
+import { initialSearchState } from 'features/search/pages/reducer'
+import { MAX_RADIUS } from 'features/search/pages/reducer.helpers'
+import { GeoCoordinates } from 'libs/geolocation'
+import { SuggestedPlace } from 'libs/place'
+import { SuggestedVenue } from 'libs/venue'
+import { mockedSuggestedVenues } from 'libs/venue/fixtures/mockedSuggestedVenues'
+import { BicolorAroundMe as AroundMe } from 'ui/svg/icons/BicolorAroundMe'
+import { BicolorEverywhere as Everywhere } from 'ui/svg/icons/BicolorEverywhere'
+import { BicolorLocationBuilding as LocationBuilding } from 'ui/svg/icons/BicolorLocationBuilding'
+import { BicolorLocationPointer as LocationPointer } from 'ui/svg/icons/BicolorLocationPointer'
+
+let mockSearchState = initialSearchState
+jest.mock('features/search/pages/SearchWrapper', () => ({
+  useSearch: () => ({
+    searchState: mockSearchState,
+  }),
+}))
+
+const DEFAULT_POSITION = { latitude: 2, longitude: 40 } as GeoCoordinates
+let mockPosition: GeoCoordinates | null = DEFAULT_POSITION
```

Comment:
Pourquoi avoir une seconde variable qui contient la même chose ? D'autant que `DEFAULT_POSITION` n'est utilisé que pour ça ?
Tu pourrais aussi bien avoir 
```suggestion
let mockPosition: GeoCoordinates | null = { latitude: 2, longitude: 40 }
```
et supprimer `DEFAULT_POSITION`